setup.py

```diff
@@ -30,12 +32,13 @@ def run(self):
 def get_tagged_version():
     process = subprocess.Popen(['git', 'describe', '--abbrev=0'], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
     out, err = process.communicate()
+    out = six.ensure_text(out)
```

Comment:
we are ensuring this is text but not ensuring the error is text.  What value does the ensure_text provide us here

Reply:
Both 'out' and 'err' are bytes-like objects, so "re.match(tag_regex, out)" gives an error on py3: "cannot use a string pattern on a bytes-like object." The 'six.ensure_text()' function just coerces the string/unicode/bytes into the str type on py3. We don't need to convert the 'err', since it's only used in '.format()', which works with byte-like objects.